packages/api/src/clickhouse/index.ts

```diff
@@ -2550,8 +2568,9 @@ export const checkAlert = async ({
     `
       SELECT 
         ?
-        count(*) as data,
-        toUnixTimestamp(toStartOfInterval(timestamp, INTERVAL ?)) as ts_bucket
+        count(*) AS data,
+        any(_string_attributes) AS attributes,
```

Comment:
I think we might also need to extract the default properties like service, level, etc. that are in columns as attributes as well.

Reply:
yeah we should. I can add those in the next PR